pyproject.toml

```
@@ -11,31 +11,36 @@ dynamic = ["version"]
 description = "A NOMAD plugin for analysis of FAIR data."
 readme = "README.md"
 authors = [
-    { name = "Sarthak Kapoor", email = "sarthak.kapoor@physik.hu-berlin.de" }
+    { name = "Sarthak Kapoor", email = "sarthakkapoor@hotmail.com" },
```

maybe this is better 

```suggestion
    { name = "Sarthak Kapoor", email = "sarthak.kapoor@physik.hu-berlin.de" },
```